src/cil.ml

```diff
@@ -4404,7 +4428,7 @@ class defaultCilPrinterClass : cilPrinter = object (self)
 
     | TArray (elemt, lo, a) ->
         (* ignore the const attribute for arrays *)
-        let a' = dropAttributes [ "const" ] a in
+        let a' = dropAttributes [ "pconst" ] a in
```

Comment:
Is it ok to only ignore `pconst` here? Or do we need to ignore `const` as well?

Reply:
This is inside CIL's printer, so `const` makes no difference because it's not used for printing anyway.